test/handlers_test.ts

```diff
@@ -30,6 +32,70 @@ const fakeOrder: SignedOrder = {
     signature: 'fakeSignature',
 };
 
+describe('parseTakerRequest', () => {
+    it('should handle an optional comparisonPrice', () => {
+        const query = {
+            sellTokenAddress: NULL_ADDRESS,
+            buyTokenAddress: NULL_ADDRESS,
+            takerAddress: NULL_ADDRESS,
+            sellAmountBaseUnits: '1225000000',
+            comparisonPrice: '320.12',
+        };
+        const request = {
+            query,
+            headers: {
+                [ZERO_EX_API_KEY_HEADER_STRING]: '0xfoo'
+            },
+
+        };
+        const parsedRequest = parseTakerRequest(request);
+        if (parsedRequest.isValid && parsedRequest.takerRequest.comparisonPrice) {
+            expect(parsedRequest.takerRequest.comparisonPrice.toNumber()).to.eql(320.12);
```

Comment:
Nit: would be nice to say `parseFloat(query.comparisonPrice)` here instead of just `320.12`.

Reply:
would `parseFloat` work on a `BigNumber`?

Reply:
Hm, good question, didn't realize bignumber was at play here.  but `query.comparisonPrice` is just a string, so i would think it should work.